### ISSUE TITLE
StackDriver logging is not working

### ISSUE THREAD

**Issue by xinoxapps** [spring-cloud/spring-cloud-gcp#1628]
Error : > Could not find an appender named [STACKDRIVER]. Did you define it below instead of above in the configuration file?

Gradle has:
```
    implementation('org.springframework.cloud:spring-cloud-gcp-starter-trace:+')
    implementation('org.springframework.cloud:spring-cloud-gcp-starter-logging:+')
```

Resources has: logback-spring.xml
```
<?xml version="1.0" encoding="UTF-8"?>
<configuration>
    <include resource="org/springframework/cloud/gcp/logging/logback-appender.xml" />
    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
    <include resource="org/springframework/boot/logging/logback/console-appender.xml" />

    <root level="INFO">
        <!-- If running in GCP, remove the CONSOLE appender otherwise logs will be duplicated. -->
        <appender-ref ref="CONSOLE"/>
        <appender-ref ref="STACKDRIVER" />
    </root>
</configuration>
```
Apparently the resource file is not included due a missing configuration step which is not mentioned in the readme.

**Comment by meltsufin** [comment]
Which version of `spring-cloud-gcp-starter-logging` are you using?
The location of `org/springframework/cloud/gcp/logging/logback-appender.xml` has recently changed in the 1.2 version.

**Comment by wbadam** [comment]
I'm experiencing the same issue with `org.springframework.cloud:spring-cloud-gcp-starter-logging:1.1.0.RELEASE` version, which is resolved using `mavenBom 'org.springframework.cloud:spring-cloud-dependencies:Greenwich.RELEASE'`.

An appender seems to exist on the path `org/springframework/cloud/gcp/autoconfigure/logging/logback-appender.xml` inside `spring-cloud-gcp-autoconfigure-1.1.0.RELEASE.jar`, but that seems to be deprecated in the source code.

**Comment by xinoxapps** [comment]
Using the latest versions:

```
implementation('org.springframework.cloud:spring-cloud-gcp-starter-trace:+')
implementation('org.springframework.cloud:spring-cloud-gcp-starter-logging:+')
```

**Comment by meltsufin** [comment]
Unless, you're using snapshots, you need to refer to the release version documentation [here](https://cloud.spring.io/spring-cloud-static/spring-cloud-gcp/1.1.0.RELEASE/multi/multi__stackdriver_logging.html).
Apologies for the confusion, since the released version doesn't have the files in the new location that is recommended for the future 1.2 release. If you're looking at the sample in `master`, it will not work against the released version as it's based on 1.2. Instead look at the sample in the 1.1.x branch: https://github.com/spring-cloud/spring-cloud-gcp/tree/1.1.x/spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample

**Comment by ShaimaaSabry** [comment]
I am facing the same error: Could not find an appender named [STACKDRIVER]

pom dependency:
```
<dependency>
    <groupId>org.springframework.cloud</groupId>
    <artifactId>spring-cloud-gcp-starter-logging</artifactId>
    <version>1.2.1.RELEASE</version>
</dependency>
```

my logback-spring.xml

<?xml version="1.0" encoding="UTF-8"?>
<configuration>
    
    <include resource="org/springframework/cloud/gcp/logging/logback-appender.xml" />
    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
    <include resource="org/springframework/boot/logging/logback/console-appender.xml" />

    <root level="INFO">
        
        
        <appender-ref ref="STACKDRIVER" />
    </root>
</configuration>
